### PR TITLE
Display devices of users

### LINKED ISSUE #1351: Display devices of users on admin panel
**Actual Behaviour**
Devices info of users is not displayed on users tab

**Expected Behaviour**
Display devices of users along with other details

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #1351 

Changes: Display devices of users

Surge Deployment Link: https://pr-1435-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:
d1

### CODE REVIEW

File: src/components/Admin/ListUser/ListUser.js [hunk 1]
```diff
@@ -255,6 +283,18 @@ export default class ListUser extends Component {
         let userList = response.users;
         let users = [];
         userList.map((data, i) => {
+          let devices = [];
+          let keys = Object.keys(data.devices);
+          keys.forEach(j => {
```
Comment (akshatnitd) [inline src/components/Admin/ListUser/ListUser.js]: Name variables correctly. `i` and `j` are not clear

Comment (PrP-11) [reply src/components/Admin/ListUser/ListUser.js]: `i` and `j` are indices commonly used during iterations.

**Comment by PrP-11** [comment]
@akshatnitd Made the changes. Please review.

**Comment by mariobehling** [comment]
Thanks. Please resolve conflicts.

**Comment by PrP-11** [comment]
@mariobehling Resolved the conflicts. Please review.

**Pull request merged**